Source/React.cpp

```diff
@@ -170,6 +182,20 @@ PeleC::react_state(
         auto const& mask = dummyMask.array(mfi);
```

Comment:
while we are at it. Let's rename this "dummyMask` to something that means something.